### PR TITLE
UCL-324 Tabs uses Quantum stylings and behaviour

### PR DESCRIPTION
**Pull request by pawel-napieracz**

### CODE REVIEW

**Comment by pawel-napieracz** [comment]
I changed `buttonBorderRadius` so maybe that's why there are so many changes to so many stories ;)

**Review by MichalGorskiOnegini — CHANGES_REQUESTED** [review]

File: src/components/Tabs/Tab.tsx
[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/Tab.tsx:1]: I don't think we should support that, as we don't have a requirement to have sub-level navigation

[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/Tab.tsx:1]: Have the same: 
Screenshot 2023-10-04 at 11 49 00
Screenshot 2023-10-04 at 11 49 12

File: src/components/Tabs/TabButton.tsx [hunk 1]
```diff
@@ -40,18 +43,22 @@ const TabButtonComponent: ForwardRefRenderFunction<HTMLButtonElement, Props> = (
     }
   }, [buttonRef.current, focused]);
 
+  const tabButtonClasses = [classes["tabbutton"]];
```
[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/TabButton.tsx:46]: You mean 
```
const tabButtonClasses = [];
tabButtonClasses.push(classes["tabbutton"])
```
?
If you want we can add it ;)

File: src/components/Tabs/Tab.tsx
[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/components/Tabs/Tab.tsx:1]: https://63e21de9a62cab5a8a3e31c7-kudmomstcz.chromatic.com/?path=/story/components-navigation-tabs--tabs-with-icons

[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/components/Tabs/Tab.tsx:1]: I mean ACTIVE state on the focus-visible.

[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/Tab.tsx:1]: Validated with design and previously with Clement. It has the same properties, but the browser renders it slightly differently.

[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/Tab.tsx:1]: Can you verify it? 
https://www.figma.com/file/zAzyG3fmBFKrdrVzFeEYAv/Quantum---2.-%F0%9F%8C%9E-Light-Components?node-id=16674%3A481731&mode=dev
https://63e21de9a62cab5a8a3e31c7-kudmomstcz.chromatic.com/?path=/story/components-navigation-tabs--tabs-with-icons

[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Tabs/Tab.tsx:1]: 🧔‍♂️, I still see that those are aligned

[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/components/Tabs/Tab.tsx:1]: looks good :)

**Review by MichalGorskiOnegini — APPROVED** [review]

**Pull request merged**